drools-ha/pom.xml

```diff
@@ -73,17 +73,6 @@
       </dependency>
 
       <!-- kafka -->
-      <dependency>
-        <groupId>org.apache.kafka</groupId>
-        <artifactId>kafka-clients</artifactId>
-        <version>${version.org.apache.kafka}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.apache.kafka</groupId>
-        <artifactId>kafka-clients</artifactId>
-        <version>${version.org.apache.kafka}</version>
-        <classifier>test</classifier>
-      </dependency>
```

Comment:
This is forcing drools-ha to use the same kafka dependency than consumer and emitter. It is done int he same jira to avoid a window of time in which several kafka client version might be colliding

Reply:
what a about just removing 
https://github.com/kiegroup/droolsjbpm-integration/blob/a95e2687e0fe78678090b8ee6dcb30f1a09dfdb6/drools-ha/pom.xml#L27 ?

and remove the version element of those dependencies ?

this modules depends on kie parent